### PR TITLE
Adding Drawer

### PR DESCRIPTION
**Pull request by LoganxDev**
Made the sidebar on the explorer page a global
drawer, small edits to comply with the linter.

Added font-awesome to get the hamburger menu icon.

### CODE REVIEW

File: src/components/AppHeader/index.js [hunk 1]
```diff
@@ -114,8 +119,8 @@ class AppHeader extends Component {
       <header className={classes.base}>
         <Grid container spacing={0}>
           <Grid item container xs={2} lg={4}>
+            <FontAwesomeIcon className={classes.hamburger} icon={faBars} size="2x" onClick={this.toggleDrawer} />
```
Comment (Hacksore) [inline src/components/AppHeader/index.js]: I would recommend just using the menu icon from material-ui rather than adding font awesome.

Thoughts?

Comment (LoganxDev) [reply src/components/AppHeader/index.js]: I definitely agree, forgot material-ui comes with those icons, good call.

**Comment by LoganxDev** [comment]
I've updated some packages because I kept getting install issues with the old ones, they were node-sass and storybook/react let me know if you have concerns about that.

**Comment by jwooning** [comment]
merged in #94

**Pull request closed without merging**